ansible/roles/azure/tasks/create_vmss.yml

```diff
@@ -0,0 +1,238 @@
+---
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
@keith-turner that's correct. The code in the file is basically the equivalent of what the existing code is doing with boto3. One addition from our side is that we are giving users the option to create a separate VM to use as proxy host instead of using the first node in the cluster.

Reply:
Ok.  All of the other ansible code is meant to be copied to the cluster and used to provision it.  This one file is very different.  It would be nice to make that difference readily apparent to anyone spelunking in the Muchos source code.  A comment would accomplish this or could put the file in a different directory tree like `$MUCHOS_HOME/ansible-client`

Reply:
In the patch to this PR, we have addressed this via. comments in the existing file. In future, we will consider the refactoring to a separate folder.

Reply:
Thanks, the comment looks great.